docs/source/developing/writing-docs.adoc

```diff
@@ -12,85 +12,156 @@ toc::[]
 
 This section contains information about contributing to the Minishift documentation.
 
-[[section-building-minishift-docs]]
-== Building Minishift documentation
+[[contribute-to-docs]]
+== Contributing to the documentation
 
-Minishift documentation is located in the `docs` sub-folder. The
+Minishift is an open-source project and we welcome contributions. Similar to code
+contributions, if you want to add or edit Minishift documentation, you can create an
+issue in our link:https://github.com/minishift/minishift/issues[Github issue tracker] and
+submit a pull request with the changes or request assistance from our technical writers.
+
+[[docs-conventions-guidelines]]
+== Conventions and guidelines
+
+The Minishift documentation is authored in the AsciiDoc markup format. Since Minishift is
+closely related to the OpenShift Origin project, we follow the same conventions for
+tagging, formatting, and structure.
+
+The documentation source structure follows a modular format, where each file is called
+a _topic_ and each topic is stored under a sub-folder based on the relevant category, such as
+getting started, using Minishift, or command reference.
+
+TIP: You can check out the raw file for this topic or other existing Minishift topics to see
+how information is structured and which tags to use.
+
+[[common-conventions]]
+=== Commonly-used conventions
+
+- Each file must contain a mandatory header section with metadata that is required for publication.
+The only exception to this is the command reference topics, which are automatically generated and
+do not require this header.
+
+- Whenever we link to internal documentation topics, you should use relative links that can
+be resolved during the publication workflow instead of hard-coding URLs.
+
+- Relative links must include breadcrumbs from the *_docs/source_* sub-folder to the topic,
+to make sure that the generated URL is correct, even if you are linking from the same sub-folder.
+
+- Because code blocks must be parsed through multiple toolchains, you should avoid complex
+or specialized source type tagging to prevent doc builds breaking. Instead, use simple four
+dashes (*----*) before and after the code block.
+
+- Each section heading must be preceded by a unique anchor ID, to help make sure that
+cross-references resolve to the correct section. The anchor ID text should match the title
+as much as possible and it does not need to include the prefix _section_ or _topic_.
+
+[[known-issues-docs]]
+=== Known issues
+
+- Use only the *link:* tag for any internal, relative, or external links. The *xref:* tag
+might cause issues with the local staging.
+
+- Currently we use the variables file to generate version numbers only. In the future we plan to expand
```

Comment:
Why is this a "known issue"? Imo this is not relevant here and the first item can be part of the link discussion a bit higher up